app/src/main/java/xterminators/spellingbee/GuiController.java

```diff
@@ -0,0 +1,126 @@
+package xterminators.spellingbee;
+
+import java.io.*;
+import java.awt.*;
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.nio.file.Paths;
+
+public class GuiController {
+
+    private JFrame m_frame;
+    private JPanel m_panel;
+    private JTextField tbGuess;
+    private UserFunctions m_functions;
```

Comment:
Is this the same UserFunctions we used the the sprint 1 CLI? If so, keep in mind that it prints to the console as output, a GUIView class might be better (if needed at all).

Reply:
This is the same UserFunctions. I thought maybe console output wouldn't matter if the user is using the GUI, but a separate class would avoid any output issues.